PWGGA/GammaConvBase/AliConvEventCuts.cxx

```diff
@@ -1374,7 +1374,9 @@ Bool_t AliConvEventCuts::SetSelectSpecialTrigger(Int_t selectSpecialTrigger)
 //     break;
   case 3:
     fSpecialTrigger=3; //specific centrality trigger selection
-    fSpecialTriggerName="AliVEvent::kCentral/kSemiCentral/kMB";
+    //fOfflineTriggerMask=AliVEvent::kINT7 | AliVEvent::kCentral | AliVEvent::kSemiCentral;
+    fTriggerSelectedManually = kTRUE;
+    fSpecialTriggerName="AliVEvent::kCentral/kSemiCentral/kINT7";
```

Comment:
@mikesas could you add here please also the kMB
otherwise the code will not work for LHC11h any longer.

Reply:
We have made sure that this does not break anything for the 2.76TeV PbPb. The cutnumber 3 is not used for this dataset.